.circleci/config.yml

```diff
@@ -72,7 +72,7 @@ workflows:
           matrix:
             parameters:
               # Run with MSRV and some modern stable Rust
-              rust-version: ["1.70.0", "1.73.0"]
+              rust-version: ["1.70.0", "1.75.0"]
```

Comment:
Not directly related to this PR, but I was wondering about this before: Is there a reason to keep the older version of clippy in CI?

Reply:
It increases out chances the whole range of clippy versions work. We saw cases in the past where older clippy versions errored for some code that newer versions accepted. So the history is not always getting more strict only.